internal/clientconn/conn.go

```diff
@@ -140,7 +146,31 @@ func (c *conn) run(ctx context.Context) (err error) {
 	}()
 
 	bufr := bufio.NewReader(c.netConn)
+
+	// if test record path is set, split netConn reader to write to file and bufr
+	if c.testRecordPath != "" {
+		if err := os.MkdirAll(c.testRecordPath, 0o755); err != nil {
+			return err
+		}
+
+		filename := fmt.Sprintf("%s_%s.bin", time.Now().Format("2006-02-01_15:04:05"), c.netConn.RemoteAddr().String())
```

Comment:
I've changed filenaming to start with a time and not address to make it more sorted.